packages/core/src/babelPlugins/babel-plugin-redwood-routes-auto-loader.ts

```diff
@@ -55,7 +56,13 @@ export default function (
                 ${availableRoutes.join('\n    ')}
               }
             }
+            `
+            .split('\n')
+            .slice(1)
+            .map((line) => line.replace('            ', ''))
```

Comment:
😆

Reply:
It's bulletproof! Will never break! 😆

Reply:
Why don't you just `.trim()`

Reply:
Another option, btw for this sort of code generation is to run it through prettier, like I'm doing here https://github.com/redwoodjs/redwood/pull/1641/files#diff-4127298228691875474bb71bfb91612ef45b7649fdb976e58bec5f5617c2d3aeR75

Reply:
I'm not `trim()`ing because I want to keep the indentation. (i.e. I don't want to remove all whitespace on all lines)

I didn't know it was that easy to use prettier programmatically! I'll give that a try, thanks 🙂

Reply:
The reason why I didn't use prettier is because this runs everytime the code is modified and increases build time.

So I wanted to do as little as possible to make it fast.

Reply:
I would love to decouple generation from compilation!